deeprankcore/query.py

```diff
@@ -219,21 +219,21 @@ def process( # pylint: disable=too-many-arguments, too-many-locals
         ) -> List[str]:
         """
         Args:
-            prefix (str, optional): Prefix for the output files. Defaults to None, which sets ./processed-queries- prefix.
-            feature_modules (List[ModuleType], optional): List of features' modules used to generate features. Each feature's module must
+            prefix (Optional[str], optional): Prefix for the output files. Defaults to None, which sets ./processed-queries- prefix.
+            feature_modules (Optional[List[ModuleType]], optional): List of features' modules used to generate features. Each feature's module must
                 implement the :py:func:`add_features` function, and features' modules can be found (or should be placed in case of a custom made feature)
                 in `deeprankcore.features` folder. Defaults to None, which means that all available modules in `deeprankcore.features` are used to generate
                 the features. 
-            cpu_count (int, optional): How many processes to be run simultaneously. Defaults to None, which takes all available cpu cores.
-            combine_output (bool, optional): For combining the .HDF5 files generated by the processes, defaults to True.
-            grid_settings (:class:`GridSettings`, optional): If valid together with `grid_map_method`, the grid data will be stored as well.
+            cpu_count (Optional[int], optional): How many processes to be run simultaneously. Defaults to None, which takes all available cpu cores.
+            combine_output (Optional[bool], optional): For combining the .HDF5 files generated by the processes. Defaults to True.
```

Comment:
Please change line 215 to `combine_output: bool = True,`
Indeed combine_output can't be actually None, but only True or False